lld/test/ELF/riscv-feature-zicfilp-func-sig.s

```diff
@@ -0,0 +1,201 @@
+# REQUIRES: riscv
+# RUN: rm -rf %t && split-file %s %t && cd %t
+# RUN: llvm-mc --filetype=obj --triple=riscv32-unknown-linux-gnu rv32-func1-zicfilp.s -o rv32-func1-zicfilp.o
```

Comment:
`-triple=riscv32` (make it clear that this is a generic ELF feature and make the triple shorter)